### PR TITLE
Improved docs for Transform Parent 

### PR DESCRIPTION
**Pull request by mrhatman**
## Description

Improved docs for Transform Parent to better describe what the component does. Also updated the Contribution docs to include --features "empty" and --features YOUR BACKEND with the change to 0.11.

## Additions

- No API additions, only docs

## Removals

- No API removals, only docs

## PR Checklist

By placing an x in the boxes I certify that I have:

- [x] Ran `cargo test --all` locally if this modified any rs files.
- [x] Ran `cargo +stable fmt --all` locally if this modified any rs files.
- [ ] Updated the content of the book if this PR would make the book outdated.
- [ ] Added a changelog entry if this will impact users, or modified more than 5 lines of Rust that wasn't a doc comment.
- [ ] Added unit tests for new APIs if any were added in this PR.
- [x] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.

### CODE REVIEW

File: amethyst_core/src/transform/components/parent.rs [hunk 1]
```diff
@@ -9,6 +9,14 @@ pub type ParentHierarchy = Hierarchy<Parent>;
 /// Component for defining a parent entity.
 ///
 /// The entity with this component *has* a parent, rather than *is* a parent.
+///
+/// If the parent entity contains a transform, then the child's transform
+/// will be applied relative to the parent's transform. For example, if a
+/// parent rotates 45 degrees around the Z axis, then the child's coordinate
+/// system will start out also rotated by 45 degrees around the Z axis.
+///
+/// If the parent entity has is own parent (and so on) then the transforms
```
Comment (azriel91) [inline amethyst_core/src/transform/components/parent.rs]: is -> its

Comment (mrhatman) [reply amethyst_core/src/transform/components/parent.rs]: Fixed

**Comment by torkleyy** [comment]
bors retry

**Comment by azriel91** [comment]
bors retry

**Comment by azriel91** [comment]
bors retry

**Pull request merged**